src/core.js

```diff
@@ -427,7 +427,7 @@ imgix.setElementImage = function (el, imgUrl) {
  * @returns {string} url of an empty image
  */
 imgix.getEmptyImage = function () {
-  return imgix.versionifyUrl('https://assets.imgix.net/pixel.gif');
+  return 'https://assets.imgix.net/pixel.gif?ixjsv=' + imgix.version;
```

Comment:
I'm not convinced that would dry this out at all. Adding the version string to this dummy URL is handled differently than building a string from a `imgix.URL` object.